util/src/main/java/com/netcracker/ConsoleHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.netcracker;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHelper {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    private ConsoleHelper(){
+        throw new RuntimeException("Don't create instance of this class");
```

Comment:
That exception looks like an overkill to me ;)
Why would anyone want to create an instance of that class inside itself (private ensures that it is the only option left) ? Only if they make it a singleton instead of a static class, but then they know what they are doing, its very unlikely that they will mess it up.